File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraLayout.Helpers
Imports DevExpress.XtraLayout
Imports System.Collections

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			ProcessItems()
		End Sub
		Private Sub ProcessItems()
			Dim items As ArrayList = MyFocusHelper.GetItemsInVisibleOrder(layoutControl1)
			For i As Integer = 0 To items.Count - 1
				Dim item As LayoutControlItem = TryCast(items(i), LayoutControlItem)
				If item IsNot Nothing Then
					item.Text = String.Format("Order = <{0}>", i)
					item.TextVisible = True
				End If
			Next i
		End Sub
	End Class


End Namespace

File: CS/Demo/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraLayout.Helpers;
using DevExpress.XtraLayout;
using System.Collections;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e)
        {
            ProcessItems();
        }
        private void ProcessItems()
        {
            ArrayList items = MyFocusHelper.GetItemsInVisibleOrder(layoutControl1);
            for (int i = 0; i < items.Count; i++)
            {
                LayoutControlItem item = items[i] as LayoutControlItem;
                if (item != null)
                {
                    item.Text = String.Format("Order = <{0}>", i);
                    item.TextVisible = true;
                }
            }
        }
    }


}